docs/developer/HootenannyConflatingANewFeatureTypeWithGenericConflation.asciidoc

```diff
@@ -28,7 +28,7 @@ See the "Hootenanny Manual Conflation" section in this document for more details
 
 First, you will want to set up a Hootenanny JSON configuration file.  An example shown here
 is similar to the file provided with Hootenanny for conflating rivers, 
-@$HOOT_HOME/conf/LinearWaterway.conf@:
+`$HOOT_HOME/conf/LinearWaterway.conf`:
```

Comment:
The @ symbol wasn't processed to anything so it was replaced everywhere to inline code.